src/rpc/server.cpp

```diff
@@ -7,7 +7,6 @@
 
 #include <fs.h>
 #include <key_io.h>
-#include <rpc/util.h>
```

Comment:
? :eyes:

Reply:
Sigh. I moved it to .h because it needed RPCHelpMan at one point. Fixed!